identity-account/src/identity/chain_state.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use serde::Serialize;
+
+use identity_iota::tangle::MessageId;
+use identity_iota::tangle::MessageIdExt;
+
+/// Holds the last published message ids of the integration and diff chains.
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct ChainState {
+  #[serde(default = "MessageId::null", skip_serializing_if = "MessageId::is_null")]
+  previous_integration_message_id: MessageId,
+  #[serde(default = "MessageId::null", skip_serializing_if = "MessageId::is_null")]
+  previous_diff_message_id: MessageId,
+}
```

Comment:
Nit: naming these just `integration_message_id` and `diff_message_id` might be fine. Very optional, not strongly opposed either way.

Reply:
I would like something that indicates that these are the ids of the messages that we have to attach the next update to, in order for it to be valid. The current naming is at least in line with the `previous_message_id` of the document, while I think a `last_` prefix would also work.

Reply:
I like `last_`! Less conflict with existing terms I think.